Book-A-Desk.Api/BookingsHttpHandler.fs

```diff
@@ -41,7 +41,9 @@ module BookingsHttpHandler =
                             User = { Email = booking.User.Email }
                         }
                     return! json output next context
-                | Error e -> return! failwith e
+                | Error e ->
```

Comment:
failwith doesn't seem to return, the request just eventually times out.

This will at least return an error.